### PR TITLE
Add new benchmarks

### PR DESCRIPTION
**Pull request by westin-m**
New benchmarks for creating/validating jwe and signedhttprequest

### CODE REVIEW

File: benchmark/Microsoft.IdentityModel.Benchmarks/Microsoft.IdentityModel.Benchmarks.csproj [hunk 1]
```diff
@@ -1,8 +1,14 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
+    <AssemblyName>Microsoft.IdentityModel.Benchmarks</AssemblyName>
```
Comment (jennyf19) [inline benchmark/Microsoft.IdentityModel.Benchmarks/Microsoft.IdentityModel.Benchmarks.csproj:4]: has this been added to the .sln so it shows up in a Benchmarks folder in VS when opening wilson.sln?

**Comment by jennyf19** [comment]
looks to be in benchmark -> Microsoft.IdentityModel.Benchmarks....can everything just be under one folder?

**Comment by jennyf19** [comment]
>         TokenValidationParameters validationParameters;

The fields should be prefixed with `_` and marked as private, same w/the other test classes.

---
Refers to: benchmark/Microsoft.IdentityModel.Benchmarks/ValidateTokenAsyncTests.cs:21 in 8425e9f. [](commit_id = 8425e9f48352283e1735d2b2c2d39d21a5f7f6d1, deletion_comment = True)

**Review by jennyf19 — APPROVED** [review]
:shipit:

**Review by jmprieur — APPROVED** [review]
LGTM
Thanks, @westin-m 

What's the perf difference between Jwt and JsonWeb?

**Comment by westin-m** [comment]
> can everything just be under one folder?

@jennyf19  Made this change, do you mind validating that it's what you were looking for?

**Pull request merged**